src/ui/public/styles/base.less

```diff
@@ -213,6 +213,9 @@ kbn-table, .kbn-table {
 
     .fa {
       font-size: 1.1em;
+      border: none;
+      background: none;
+      padding: 0;
```

Comment:
Let's create a new class called `.docTableHeaderButton` in `doc_table/doc_table.less` and move these styles there, just so they're a little more organized together and semantically specific.